server/repository/src/db_diesel/name_store_join.rs

```diff
@@ -47,6 +47,7 @@ type NameStoreJoins = (NameStoreJoinRow, (NameLinkRow, NameRow));
 pub struct NameStoreJoinFilter {
     pub id: Option<EqualFilter<String>>,
     pub name_id: Option<EqualFilter<String>>,
+    pub store_id: Option<EqualFilter<String>>,
```

Comment:
I think i found potential future bug, when patient is added we only add name_store_join if one with that name exists, probably ok for remote sites but future uses of create_patient_name_store_join might have come with bugs

Reply:
I'm not sure I understand?